### PR TITLE
improve error message in case policy module could not be loaded

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- fixes https://github.com/RasaHQ/rasa_core/issues/1576
- fixes https://github.com/RasaHQ/rasa_core/issues/1575
- raise `InvalidPolicyConfig` error in case the policy in the module could not be loaded, the `policies` key is empty or not there

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- ~[ ] updated the documentation~
- [x] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
`added this to the 0.13 release milestone 30 minutes ago` @tmbo sneaky

File: rasa_core/policies/ensemble.py [hunk 1]
```diff
@@ -217,10 +217,17 @@ def load(cls, path: Text) -> 'PolicyEnsemble':
 
     @classmethod
     def from_dict(cls, dictionary: Dict[Text, Any]) -> List[Policy]:
+        policies = dictionary.get('policies')
```
Comment (akelad) [inline rasa_core/policies/ensemble.py]: can we also allow the word "policy" please?

File: rasa_core/policies/ensemble.py [hunk 2]
```diff
@@ -217,10 +217,17 @@ def load(cls, path: Text) -> 'PolicyEnsemble':
...
+        if policies is None:
+            raise InvalidPolicyConfig("The policy configuration file has to "
+                                      "include a key 'policies'.")
```
Comment (akelad) [inline rasa_core/policies/ensemble.py]: Maybe make this a little easier to understand with e.g.  "You didn't define any policies, please define them under `policies:` in your yaml file"

File: rasa_core/policies/ensemble.py [hunk 3]
```diff
@@ -242,11 +249,16 @@ def from_dict(cls, dictionary: Dict[Text, Any]) -> List[Policy]:
...
-            policy_object = constr_func(**policy)
 
-            policies.append(policy_object)
+            if constr_func:
+                policy_object = constr_func(**policy)
+                parsed_policies.append(policy_object)
+            else:
+                raise InvalidPolicyConfig("Module for policy '{}' could not be "
+                                          "loaded. Please make sure the name "
+                                          "is valid.".format(policy_name))
```
Comment (akelad) [inline rasa_core/policies/ensemble.py]:
```suggestion
                                          "is a valid policy.".format(policy_name))
```

**Comment by wochinge** [comment]
@akelad Ready for another review

**Comment by akelad** [comment]
@wochinge for future reference, if a PR is approved with minor changes, that means you can merge it without another review after you've made the minor changes :) only if the reviewer hits "request changes", then they need to review it again

**Comment by akelad** [comment]
in other words, feel free to merge once the build passes

**Comment by tmbo** [comment]
I will do the honours then :wink:

**Pull request merged**